Mathlib/Data/Multiset/Basic.lean

```diff
@@ -14,9 +14,20 @@ import Mathlib.Order.Hom.Basic
 
 /-!
 # Multisets
-These are implemented as the quotient of a list by permutations.
+
+Multisets are finite sets with duplicates allowed. They are implemented here as the quotient of
+lists by permutation. This gives them computational content.
+
 ## Notation
-We define the global infix notation `::ₘ` for `Multiset.cons`.
+
+* `0`: The empty multiset.
+* `{a}`: The multiset containing a single occurrence of `a`.
+* `a ::ₘ s`: The multiset containing one more occurrence of `a` than `s`.
```

Comment:
```suggestion
* `a ::ₘ s`: The multiset containing one more occurrence of `a` than `s` has.
```
(else a valid interpretation would be that the multiplicity of `a` is one more than the multiplicity of `s`

Reply:
What about now? Not sure I am speaking the english very good